cybershell/benchmarking_framework.py

```
@@ -0,0 +1,791 @@
+            async with semaphore:
+                return await self._run_single_benchmark(target)
+        
+        tasks = [run_with_semaphore(target) for target in targets]
+        results = await asyncio.gather(*tasks)
+        
+        return results
+    
+    async def _run_single_benchmark(self, target: BenchmarkTarget) -> BenchmarkResult:
+        """Run a single benchmark test"""
+        
+        start_time = datetime.now()
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)